cmake/f3dPlugin.cmake

```diff
@@ -263,4 +283,23 @@ macro(f3d_plugin_build)
       EXCLUDE_FROM_ALL)
   endforeach()
 
+  # JSON
+  string(JSON F3D_PLUGIN_JSON
+    SET "${F3D_PLUGIN_JSON}" "name" "\"${F3D_PLUGIN_NAME}\"")
+
+  string(JSON F3D_PLUGIN_JSON
+    SET "${F3D_PLUGIN_JSON}" "description" "\"${F3D_PLUGIN_DESCRIPTION}\"")
+
+  string(JSON F3D_PLUGIN_JSON
+    SET "${F3D_PLUGIN_JSON}" "version" "\"${F3D_PLUGIN_VERSION}\"")
+
+  string(JSON F3D_PLUGIN_JSON
+    SET "${F3D_PLUGIN_JSON}" "type" "\"${F3D_PLUGIN_TYPE}\"")
+
+  file(WRITE "${CMAKE_BINARY_DIR}/${F3D_PLUGIN_NAME}.json" "${F3D_PLUGIN_JSON}")
+
+  install(FILES "${CMAKE_BINARY_DIR}/${F3D_PLUGIN_NAME}.json"
+      DESTINATION "share/f3d/plugins"
```

Comment:
for now share/f3d/ contains only config file. Should we have a subdir for that too ?

Reply:
config file is in /etc, not {prefix}/share/f3d

Reply:
/etc/ is for a package manager install only. BInary package ship config files to /usr/share/f3d

https://f3d.app/doc/user/CONFIGURATION_FILE.html

Reply:
I think we'll have `config.d` at some point, so let's keep config.json without subdirectory for the moment